src/index.js

```diff
@@ -92,7 +92,7 @@ function ParseServer({
 }) {
   
   // Initialize the node client SDK automatically
-  Parse.initialize(appId, javascriptKey, masterKey);
+  Parse.initialize(appId, javascriptKey || 'unused', masterKey);
```

Comment:
That was just me mimicking the previous behaviour when updating this function to ES6 style